### PR TITLE
Move administration page to proper location

### PR DESCRIPTION
**Pull request by sarjon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | It moves administration page related code under /Configure/AdvancedParameters to keep code organization the same across all migrated pages.
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | n/a
| How to test?  | Administration page should work the same.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8979)

### CODE REVIEW

File: src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Administration/FormHandler.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-namespace PrestaShopBundle\Form\Admin\AdvancedParameters\Administration;
+namespace PrestaShopBundle\Form\Admin\Configure\AdvancedParameters\Administration;
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Administration/FormHandler.php]: gosh, the pull request about Form Handling is still not merged, this making me so crazy :/

**Comment by mickaelandrieu** [comment]
@sarjon you can rebase and use new Form Handler system  \o/

**Comment by sarjon** [comment]
@mickaelandrieu done

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Thank you @sarjon !

**Pull request merged**